ozaria/site/views/play/level/tome/SpellView.coffee

```diff
@@ -733,13 +738,14 @@ module.exports = class SpellView extends CocoView
     @aceDoc.removeListener 'change', @onCodeChangeMetaHandler if @onCodeChangeMetaHandler
     onSignificantChange = []
     onAnyChange = [
-      _.debounce @updateAether, if @options.level.isType('game-dev') then 10 else 500
       _.debounce @notifyEditingEnded, 1000
       _.throttle @notifyEditingBegan, 250
       _.throttle @notifySpellChanged, 300
       _.throttle @updateLines, 500
       _.throttle @hideProblemAlert, 500
     ]
+    unless @options.level.isType('game-dev')
+      onAnyChange.push(_.debounce(@updateAether, 500))
```

Comment:
Why is this not required for game-dev/capstone? Also do we want to do this for all game-dev levels or just capstone, or probably its the same thing(i.e. only capstones can be game-dev)? I am not sure, but might be worth checking.

Reply:
I don't know of game dev levels that are not capstone now, but in case we add that in the future I will add an additional check. 

This change means that the level is not constantly updated by the code changing, which is the purpose of this PR.